packages/flutter_driver/lib/src/driver/driver.dart

```diff
@@ -571,6 +571,19 @@ class FlutterDriver {
     return result.changedState;
   }
 
+  /// Retrieves the semantics node id for the object returned by [finder], or
```

Comment:
I believe arguments mentioned in doc comments should be enclosed in `` instead of [] unless the argument is also a field on the class.

(here and below)

Reply:
Done